model/raw/src/main/scala/aqua/raw/ops/RawTag.scala

```diff
@@ -290,13 +290,17 @@ case class ClosureTag(
   override def usesVarNames: Set[String] = Set.empty
 
   override def renameExports(map: Map[String, String]): RawTag =
-    copy(func = func.copy(name = map.getOrElse(func.name, func.name)))
+    copy(func =
+      func.copy(
+        name = map.getOrElse(func.name, func.name)
+      )
+    )
 
   override def mapValues(f: ValueRaw => ValueRaw): RawTag =
     copy(
       func.copy(arrow =
         func.arrow.copy(
-          ret = func.arrow.ret.map(_.mapValues(f)),
+          ret = func.arrow.ret.map(_.map(f)),
```

Comment:
Why is this needed? Semantic of `mapValues` is "map your *immediate* desecndants", using `map` here maps the whole subtree of every return values. If one wants to map the whole subtree for each children, `.mapValues(_.map(f))` could be used on *call site*